rules/linux/persistence_linux_shell_activity_by_web_server.toml

```
@@ -67,16 +67,19 @@ references = [
     "https://www.elastic.co/security-labs/elastic-response-to-the-the-spring4shell-vulnerability-cve-2022-22965",
 ]
 risk_score = 47
-rule_id = "231876e7-4d1f-4d63-a47c-47dd1acdc1cb"
+rule_id = "b7b2c320-d4db-4f8e-8f92-83f9d0c3e6a4"
 severity = "medium"
 tags = ["Elastic", "Host", "Linux", "Threat Detection", "Persistence", "Investigation Guide", "Elastic Endgame"]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and event.type:(start or process_started) and
-process.name:(bash or dash or ash or zsh or "python*" or "perl*" or "php*") and
-process.parent.name:("apache" or "nginx" or "www" or "apache2" or "httpd" or "www-data")
+process where event.type == "start" and
+process.args:("whoami*", "id", "uname*", "cat*", "hostname*", "ip*", "curl*", "wget*", "pwd") and
+process.name : ("bash", "sh", "ash", "dash", "zsh", "rbash", "csh", "ksh", "tmux", "python*", "perl*", "php*") and
```

```suggestion
process.name : ("bash", "sh", "ash", "dash", "zsh", "rbash", "csh", "tcsh", "ksh", "tmux", "python*", "perl*", "php*") and
```
Nice additions, added one more